.envrc

```diff
@@ -0,0 +1 @@
+use flake
```

Comment:
I think adding a `flake.nix` file is fine, but prescribing it for all direnv users I'm skeptical about.

Reply:
I'd remove the `.envrc` file and add it to `.gitignore` too. That way people can choose to use it or not.

Reply:
standard practice is: change it to `.envrc.sample` and ask users to `mv  .envrc.sample .envrc` in setup instruction for nix (and with .gitignore)

Reply:
I'd opt with just removing it since it's a single line, and providing a simple will force to pick between flake and non-flake usage.